openpype/pipeline/__init__.py

```diff
@@ -8,6 +8,14 @@
 )
 from .anatomy import Anatomy
 
+from .colorspace import (
```

Comment:
Content of `colorspace` requires specific dependencies thus should not be in `pipeline/__init__.py`.

Reply:
This is solved in https://github.com/pypeclub/OpenPype/pull/4270 by bypassing it via subprocessing script file.